AK/Endian.h

```diff
@@ -125,11 +125,21 @@ struct Traits<LittleEndian<T>> : public GenericTraits<LittleEndian<T>> {
     static constexpr bool is_trivially_serializable() { return Traits<T>::is_trivially_serializable(); }
 };
 
+template<typename T>
+struct Traits<LittleEndian<T> const> : public GenericTraits<LittleEndian<T> const> {
+    static constexpr bool is_trivially_serializable() { return Traits<T>::is_trivially_serializable(); }
```

Comment:
Maybe this should be `Traits<T const>`? Same for BigEndian.

Reply:
`T` itself isn't what's `const` here so I don't think we need that? i.e. I'd expect we need that if we were defining `Traits<LittleEndian<T const>>`.

Reply:
Ah you're right! Ignore me. :laughing: